potassium/__init__.py

```diff
@@ -1,3 +1,4 @@
 from .potassium import *
 from .hooks import *
-from .store import Store, RedisConfig
\ No newline at end of file
+from .store import Store, RedisConfig
+from .types import Request, Response
```

Comment:
style nit: absolute > relative, apply to all - not sure what banana style guide is like.